server/actorController.js

```diff
@@ -3,14 +3,10 @@ const playData = require('./playData');
 
 const actorController = {
   newActor: (req, res, next) => {
-    const { firstName, lastName } = req.body;
     // add actor to the database
-    const values = [firstName.trim(), lastName.trim()];
-    const text = `INSERT INTO actors 
-          (first_name, last_name)
-          VALUES ($1, $2)`;
+    const values = [req.body.name.trim()];
```

Comment:
I'm a bit unsure too, and haven't seen other recent react codebases. Probably does make a bit more sense to destructure since that follows the pattern used in all the other endpoints we have.